Cesium.CodeGen/Contexts/AssemblyContext.cs

```diff
@@ -101,6 +107,41 @@ public AssemblyDefinition VerifyAndGetAssembly()
         {
             GlobalType = Module.GetType("<Module>");
         }
+        _globalTypeStaticCtor = new(
+            () =>
+            {
+                var ctor = BuildStaticCtor();
+                GlobalType.Methods.Add(ctor);
+                return ctor;
+            });
+    }
+
+    private MethodDefinition BuildStaticCtor()
+    {
+        var methodAttributes = MethodAttributes.Private
+                               | MethodAttributes.HideBySig
+                               | MethodAttributes.SpecialName
+                               | MethodAttributes.RTSpecialName
+                               | MethodAttributes.Static;
+        MethodDefinition method = new MethodDefinition(".cctor", methodAttributes, Module.TypeSystem.Void);
+        method.Body.Instructions.Add(Instruction.Create(OpCodes.Ret));
+        return method;
+    }
+
+    internal void AddFieldInitialization(FieldDefinition field, Ir.Expressions.IExpression value)
+    {
+        var lval = new Ir.Expressions.LValues.LValueGlobalVariable(field);
+        var context = new TranslationUnitContext(this);
+        var scope = new FunctionScope(context, GlobalTypeStaticCtor);
+        var lastOpcode = GlobalTypeStaticCtor.Body.Instructions.Last();
+        if (lastOpcode.OpCode == OpCodes.Ret)
```

Comment:
This code is very strange. How are you even getting this `Ret`? I can't see who puts it there.

Reply:
line 127

Reply:
Then remove it :)